From 364ebd75c2247359f7c48320dfbfbc85a284a725 Mon Sep 17 00:00:00 2001
From: Eric Yang <eric.yang2@amd.com>
Date: Mon, 22 Feb 2016 13:36:26 -0500
Subject: [PATCH 0826/1110] drm/amd/dal: address missed change from ipp
 refactor

The degamma mode enum was refactored, the usage wasn't
change accordingly, causing asserts

Signed-off-by: Eric Yang <eric.yang2@amd.com>
Reviewed-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce110/dce110_ipp_gamma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_ipp_gamma.c b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_ipp_gamma.c
index fb90a6a..5d0fc59 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_ipp_gamma.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_ipp_gamma.c
@@ -67,7 +67,7 @@ bool dce110_ipp_set_degamma(
 	uint32_t degamma_type = (mode == IPP_DEGAMMA_MODE_HW_sRGB) ? 1 : 0;
 
 	ASSERT(mode == IPP_DEGAMMA_MODE_BYPASS ||
-			mode == IPP_DEGAMMA_MODE_USER_PWL);
+			mode == IPP_DEGAMMA_MODE_HW_sRGB);
 
 	set_reg_field_value(
 		value,
-- 
2.7.4

